Added sentry instructions

Parth4337 opened the pull request:
Added instructions for setting up sentry structure and fixed some typos

Parth4337 committed:
Added sentry insturctions

Added instructions for setting up sentry structure and fixed some typos

dzmitryhil commented on docs/testnet/onomy-testnet-docs/setting-up-a-validator.md:
````diff
@@ -153,7 +158,7 @@ onomyd --home $HOME/.onomy tx bank send <your validator address> <your delegate
 2. Using faucet command, from the Onomy-side faucet:
 ```
-curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"10000000nom\"  ]}"
+curl -X POST http://testnet1.onomy.io:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"<your delegate cosmos address>\",  \"coins\": [    \"100000000nom\"  ]}"
````
For the next testnet the main currency will be "anom", so we need to change it for all scripts and docs

Parth4337 replied:
Changed, Check once

dzmitryhil commented on docs/testnet/onomy-testnet-docs/pre-installation.md:
```diff
@@ -26,24 +25,26 @@ export PATH=$PATH:$HOME/.onomy/bin
 `
```
Check markdown link check it is red
https://github.com/onomyprotocol/onomy/runs/4279388673?check_suite_focus=true

Parth4337 replied:
Done

Parth4337 committed:
Merge branch 'onomyprotocol:dev' into dev

Parth4337 committed:
Changed "external_address' parameter in config.toml

Changed scripts to add machine's ip rather than 0.0.0.0 to 'external_addresses' parameter in config.toml

Parth4337 committed:
Fixed broken md link

Fixed broken link in readme.md

Parth4337 committed:
fixed mistake in ip command

Fixed IP Command behaviour read command flags

Parth4337 committed:
Changed NOM to ANOM

changed all the mentions of NOM to ANOM in docs

Parth4337 committed:
Changed NOM to ANOM in scripts

changed all the mentions of NOM to ANOM in deployment scripts

cgdusek commented:
Why are these test indicator boxes red and I see red X ever commit?
